coreneuron/mechanism/patternstim.cpp

```diff
@@ -199,7 +201,7 @@ Point_process* nrn_artcell_instantiate(const char* mechname) {
     int psize = corenrn.get_prop_param_size()[type];
     int dsize = corenrn.get_prop_dparam_size()[type];
     // int layout = nrn_mech_data_layout[type]; // not needed because singleton
-    Memb_list* ml = tml->ml;
+    Memb_list* ml = nt->tml->ml;
```

Comment:
Just for the record : this and previous change is not doing same thing as old code (`nt->tml->ml` is always first element here).

Reply:
Uff, good catch!